configs/HGLRCF722AIOV2/config.h

```diff
@@ -0,0 +1,124 @@
+#define USE_FLASH_W25Q128FV
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_MAX7456
+
+#define BEEPER_PIN                      PC13
+#define MOTOR1_PIN                      PB4
+#define MOTOR2_PIN                      PB5
+#define MOTOR3_PIN                      PB0
+#define MOTOR4_PIN                      PB1
+#define MOTOR5_PIN                      PA15
+#define MOTOR6_PIN                      PB3
+#define MOTOR7_PIN                      PB6
+#define MOTOR8_PIN                      PB7
+#define RX_PPM_PIN                      PA3
+#define RX_PWM1_PIN                     PA2
+#define RX_PWM2_PIN                     PA1
+#define RX_PWM3_PIN                     PA0
+#define LED_STRIP_PIN                   PA8
+#define UART1_TX_PIN                    PA9
+#define UART2_TX_PIN                    PA2
+#define UART3_TX_PIN                    PB10
+#define UART4_TX_PIN                    PA0
+#define UART5_TX_PIN                    PC12
+#define UART6_TX_PIN                    PC6
+#define UART1_RX_PIN                    PA10
+#define UART2_RX_PIN                    PA3
+#define UART3_RX_PIN                    PB11
+#define UART4_RX_PIN                    PA1
+#define UART5_RX_PIN                    PD2
+#define UART6_RX_PIN                    PC7
+#define I2C1_SCL_PIN                    PB8
+#define I2C1_SDA_PIN                    PB9
+#define LED0_PIN                        PA13
+#define LED1_PIN                        PA14
+#define SPI1_SCK_PIN                    PA5
+#define SPI2_SCK_PIN                    PB13
+#define SPI1_SDI_PIN                    PA6
+#define SPI2_SDI_PIN                    PB14
+#define SPI1_SDO_PIN                    PA7
+#define SPI2_SDO_PIN                    PB15
+#define ADC_VBAT_PIN                    PC2
+#define ADC_RSSI_PIN                    PC0
+#define ADC_CURR_PIN                    PC1
+#define BARO_CS_PIN                     PA4
+#define PINIO1_PIN                      PC8
+#define PINIO2_PIN                      PC9
+#define FLASH_CS_PIN                    PC10
+#define MAX7456_SPI_CS_PIN              PB12
+#define GYRO_1_EXTI_PIN                 PC4
+#define GYRO_1_CS_PIN                   PB2
+#define USB_DETECT_PIN                  PC14
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PB4,  1,  0) \
+    TIMER_PIN_MAP( 1, PB5,  1,  0) \
+    TIMER_PIN_MAP( 2, PB0,  2,  0) \
+    TIMER_PIN_MAP( 3, PB1,  2,  0) \
+    TIMER_PIN_MAP( 4, PA15, 1,  0) \
+    TIMER_PIN_MAP( 5, PB3,  1,  0) \
+    TIMER_PIN_MAP( 6, PB6,  1,  0) \
+    TIMER_PIN_MAP( 7, PB7,  1,  0) \
+    TIMER_PIN_MAP( 8, PA8,  1,  2) \
+    TIMER_PIN_MAP( 9, PA3,  3, -1) \
+    TIMER_PIN_MAP(10, PA2,  3, -1) \
+    TIMER_PIN_MAP( 9, PA1,  2,  0) \
+    TIMER_PIN_MAP(10, PA0,  2,  0)
+
+#define ADC1_DMA_OPT                    1
+
+#define MAG_I2C_INSTANCE                I2CDEV_1
+#define BARO_I2C_INSTANCE               I2CDEV_1
+#define DEFAULT_DSHOT_BURST             DSHOT_DMAR_ON
```

Comment:
F722 should use bitbang by default
```suggestion
#define DEFAULT_DSHOT_BITBANG         DSHOT_BITBANG_ON
```